.jenkins

```diff
@@ -385,6 +385,63 @@ pipeline {
                         }
                     }
                 }
+
+                stage('SYCL') {
+                    agent {
+                        dockerfile {
+                            filename "Dockerfile.sycl"
+                            dir "docker"
+                            args '-v /tmp/ccache.kokkos:/tmp/ccache'
+                            label 'NVIDIA_Tesla_V100-PCIE-32GB && nvidia-docker'
+                        }
+                    }
+                    steps {
+                        sh 'ccache --zero-stats'
+                        sh 'rm -rf build && mkdir -p build'
+                        dir('build') {
+                            sh '''
+                                cmake \
+                                    -D CMAKE_INSTALL_PREFIX=$ARBORX_DIR \
+                                    -D CMAKE_BUILD_TYPE=Release \
+                                    -D CMAKE_CXX_COMPILER=clang++ \
+                                    -D CMAKE_CXX_EXTENSIONS=OFF \
```

Comment:
Don't you need to set the standard to C++17?

Reply:
The compiler doesn't need any flag to provide C++17 but I could add it.